lib/exercises.rb

```diff
@@ -2,19 +2,49 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n*m)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
This works but your time complexity is O(n * m log m) where n is the number of strings and m is the length of the strings since you are sorting.

If the strings are guaranteed to be within a certain length, you could drop the m term and it would be O(n).